lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -61,12 +61,16 @@ defmodule ExUnit.DocTest do
       2
 
   If you don't want to assert for every result in a doctest, you can omit
-  the result:
+  the result in the middle of expressions:
 
       iex> pid = spawn(fn -> :ok end)
       iex> is_pid(pid)
       true
 
+  As well as at the end:
+
+      iex> Mod.do_a_call_that_should_not_raise!(...)
```

Comment:
I think if the result can be omitted in between expressions and in the last one, then they can be omitted altogether.